lib/cryptoexchange/exchanges/buyucoin/services/market.rb

```diff
@@ -4,29 +4,17 @@ module Services
       class Market < Cryptoexchange::Services::Market
         class << self
           def supports_individual_ticker_query?
-            false
+            true
```

Comment:
Could use the `false` case with https://api.buyucoin.com/ticker/v1.0/liveData ?

Reply:
yes, we can definitely because initially, I saw they able to provide individual data so it might be better to just use it instead of querying them all.

Reply:
batch query is good, because we do store them in cache. 50 ticker calls = 1 request to the api.